tests/phpunit/integration/Core/Authentication/Has_Multiple_AdminsTest.php

```diff
@@ -46,7 +47,64 @@ public function test_get() {
 		$this->factory()->user->create( array( 'role' => 'administrator' ) );
 
 		// Delete the transient and check that we have multiple admins now.
-		$this->transients->delete( Has_Multiple_Admins::OPTION );
 		$this->assertTrue( $has_multiple_admins->get(), 'Should return true when there are multiple admin users' );
 	}
+
+	public function test_register__user_register_hook() {
+		$has_multiple_admins = new Has_Multiple_Admins( $this->transients );
+		$has_multiple_admins->register();
+
+		// Initially, there is only one admin (ID=1).
```

Comment:
It would be good to demonstrate this with a query, or perhaps delete the initial user to start with so that we essentially prove/verify this rather than assume it. All the DB changes will be rolled back after the test, so either should be safe.